symbolic-debuginfo/src/macho/bcsymbolmap.rs

```diff
@@ -194,6 +198,201 @@ impl<'a, 'd> Iterator for BcSymbolMapIterator<'a, 'd> {
 
 impl FusedIterator for BcSymbolMapIterator<'_, '_> {}
 
+/// Error type when parsing an Apple PropertyList into [`BitcodeUuidMapping`].
+#[derive(Debug, Error)]
+#[error("{kind}")]
+pub struct PListError {
+    kind: PListErrorKind,
+    #[source]
+    source: Option<Box<dyn Error + Send + Sync + 'static>>,
+}
+
+impl PListError {
+    /// The kind of error, giving a little more detail.
```

Comment:
Standard documentation for this is:

```rust
/// Returns the corresponding [`PListErrorKind`] for this error.
```

Inspired from `std::io::Error::kind` and used consistently throughout symbolic.